src/features/CastAndCrew/index.js

```diff
@@ -1,130 +1,93 @@
+import React, { useState, useEffect } from "react";
+import { getMovieCredits } from "../movieDetailsAPI"; // Upewnij się, że import jest poprawny
```

Comment:
Comments should be deleted before merging. Unless you think otherwise and they will be helpfull later on.

Reply:
I will fix it.